presto-main/src/main/java/com/facebook/presto/sql/gen/JoinProbeCompiler.java

```diff
@@ -250,6 +252,23 @@ private static void generateConstructor(ClassDefinition classDefinition,
         constructor.comment("this.positionCount = page.getPositionCount();")
                 .append(thisVariable.setField(positionCountField, page.invoke("getPositionCount", int.class)));
 
+        constructor.comment("this.probeOutputChannels = new int[<probeOutputChannelCount>];");
+        constructor
+                .append(thisVariable)
+                .push(probeOutputChannels.size())
+                .append(newPrimitiveArray(type(int.class)))
+                .putField(probeOutputChannelsField)
+                .append(thisVariable)
```

Comment:
you could insert another `.comment(..)` here (the previous one got just fulfilled)